t/plugin/error-log-logger-clickhouse.t

```diff
@@ -98,7 +98,7 @@ done
 --- error_log
 this is a warning message for test2
 clickhouse body: INSERT INTO t FORMAT JSONEachRow
-clickhouse headers: x-clickhouse-key:dpwomMlEsHH2L7wSUi6YiQ==
+clickhouse headers: x-clickhouse-key:a
```

Comment:
why change this?

Reply:
https://github.com/apache/apisix/pull/11076 
The modification of error-log-logger-clickhouse.t in this PR is wrong. The header sent to clickhouse should be decrypted.